ompi/mca/pml/ucx/pml_ucx.c

```diff
@@ -492,11 +493,14 @@ int mca_pml_ucx_recv(void *buf, size_t count, ompi_datatype_t *datatype, int src
                               mca_pml_ucx_get_datatype(datatype),
                               ucp_tag, ucp_tag_mask, req);
 
-    for (;;) {
-        status = ucp_request_test(req, &info);
-        if (status != UCS_INPROGRESS) {
-            mca_pml_ucx_set_recv_status_safe(mpi_status, status, &info);
-            return OMPI_SUCCESS;
+    while (1) {
```

Comment:
can't we reuse MCA_COMMON_UCX_WAIT_LOOP?

Reply:
added macro for progress loop